process/graduation-proposal-template.md

```diff
@@ -1,25 +1,212 @@
+1. License type (should be Apache 2): ???
+1. URL to license:
+   * ???
+1. TOC sponsor name (if one exists): ???
+1. URL to Core Infrastructure Initiative
+   [Best Practices Badge](https://bestpractices.coreinfrastructure.org/):
+   * ???
+1. URLs to required reviews:
+   * Independent and 3rd party security audit results:
+     * ???
+1. URLs to governance documentation:
+   * ???
+1. Active maintainers:
+   * URL to document listing current active maintainers:
+     * ???
+   * Organizations with active maintainers as of the time of this PR (should
+     have at least 2 orgs):
+     * ???org (???#ofMaintainersFromOrg)
+1. URLs to previous CNCF applications (e.g. sandbox, incubator):
+   * sandbox: ???
+   * incubator: ???
 
-### * Have achieved and maintained a [Core Infrastructure Initiative Best Practices Badge](https://bestpractices.coreinfrastructure.org/).
 
-### * Have completed an independent and third party security audit with results published of similar scope and quality as [this example](https://github.com/envoyproxy/envoy#security-audit) which includes all critical vulnerabilities and all critical vulnerabilities need to be addressed before graduation.
+## Graduation Information
 
-### * Explicitly define a project governance and committer process. The committer process should cover the full committer lifecycle including onboarding and offboarding or emeritus criteria. This preferably is laid out in a GOVERNANCE.md file and references an OWNERS.md file showing the current and emeritus committers.
+1. Why is this project ready to graduate?
+   * ???
 
-### * Explicitly define the criteria, process and offboarding or emeritus conditions for project maintainers; or those who may interact with the CNCF on behalf of the project. The list of maintainers should be preferably be stored in a MAINTAINERS.md file and audited at a minimum of an annual cadence.
+1. What has changed since the project was approved for incubation?
+   * ???
 
-### * Have a public list of Project adopters for at least the primary repo (e.g., ADOPTERS.md or logos on the Project website). For a specification, have a list of adopters for the implementation(s) of the spec. Refer to [FAQs](https://github.com/cncf/toc/blob/main/FAQ.md#what-is-the-definition-of-an-adopter) for guidelines on identifying adopters.
+1. How can the project survive its founders and founding companies?
+   * ???
 
-## Incubation Details
-_**Project should address each area listed below**_
+1. Project adopters:
+   * Typical adopters (e.g. cloud providers, end users):
+     * ???
+   * Provide a list of adopters who are willing to chat with the TOC about
+     their experiences with the project and its community (about 2-4. Questions
+     might include: how/why do you use it? its stengths and weaknesses?
+     are community interactions positive/responsive/timely?):
+     * ???name (???id at ???domainName)
+   * Adopter quotes with attribution:
+     * ???name : ???quote
+
+
+## Due Diligence
```

Comment:
I disagree with the need for any due diligence at the graduation phase, unless there is any suggestion that a project may have regressed since entering the CNCF and may not continue to meet the incubation criteria.  That process could well happen with an incubating project and should be addressed by project review and/or the archival process, and need not be duplicated by graduation.

I take this position based on the expressed ideals that "[[the] incubation level is the point at which we expect to perform full due diligence on projects](https://github.com/cncf/toc/blob/main/process/graduation_criteria.md)", and that the stated intention of graduation is [a signal to the market that the software is ready for the early majority, not just early adopters](https://www.cncf.io/projects/#maturity-levels).

An exception would be that if a project intends to _join_ at Graduated, which has not yet happened but is explicitly mentioned as a possibility.  In that case, I would hope that the processes for "join at Incubation" and "demonstrate readiness for Graduation" were non-duplicative and could just both be followed.

Reply:
I'm ok with removing it if the TOC would like that. I included it because:
1 - as you suggested, a project may want to join the CNCF at the graduated level
2 - since part of my goal is to reduce the amount of time for the TOC to review any pertinent information about a project, I wanted to put everything in one spot, even if that info was originally provided at a lower stage.
3 - worst case people could just copy the info from the info previously provided

I'll leave it to the TOC to decide if including this here is useful or not - I'll do whatever speeds up their graduation review process.